src/IssueViz.Security/HotspotsList/HotspotsControl.xaml

```diff
@@ -252,12 +256,23 @@
                     CommandParameter="{Binding Hotspots/}"/>
             </DataGrid.InputBindings>
             <DataGrid.Columns>
-                <DataGridTemplateColumn CanUserSort="False" Width="25">
+                <DataGridTemplateColumn CanUserSort="False" CanUserResize="False" Width="30">
+                    <DataGridTemplateColumn.HeaderStyle>
+                        <Style TargetType="DataGridColumnHeader" BasedOn="{StaticResource {x:Type DataGridColumnHeader}}">
+                            <Setter Property="ToolTipService.ToolTip" Value="Location navigability status" />
+                        </Style>
+                    </DataGridTemplateColumn.HeaderStyle>
                     <DataGridTemplateColumn.CellTemplate>
                         <DataTemplate>
-                            <vsImaging:CrispImage Height="16" Width="16"
-                                                  Moniker="{x:Static vsCatalog:KnownMonikers.DocumentWarning}" 
-                                                  Visibility="{Binding Hotspot.Span, Converter={StaticResource SpanToInvertedVisibilityConverter}, Mode=OneWay}"/>
+                            <Grid>
+                                <vsImaging:CrispImage Height="16" Width="16"
+                                                      Moniker="{x:Static vsCatalog:KnownMonikers.DocumentWarning}" 
+                                                      Visibility="{Binding Hotspot.Span, Converter={StaticResource SpanToInvertedVisibilityConverter}, Mode=OneWay}"/>
+                                <vsImaging:CrispImage Height="16" Width="16"
+                                                      ToolTipService.ToolTip="Location is navigable. Use double-click or Enter to open the document."
```

Comment:
Note that the other image doesn't have an explicit tooltip.
This is because when a location is not navigable, there is a tooltip on the entire row (set by style https://github.com/SonarSource/sonarlint-visualstudio/blob/6a322aaa3d75612961a0f9077e65fd0b2853e2c9/src/IssueViz.Security/HotspotsList/HotspotsControl.xaml#L131)